tests/operations.go

```diff
@@ -870,3 +947,21 @@ func (ops *Operations) GetBDCCount(lSelector, namespace string) int {
 	Expect(err).ShouldNot(HaveOccurred())
 	return len(bdcList.Items)
 }
+
+// GetCSPCBDListForNode returns unclaimed block devices that can be used.
+func (ops *Operations)GetCSPCBDListForNode(nodeName string, blockDeviceCount int) []*apis.CStorPoolClusterBlockDevice {
+	bdList,err:=ops.BDClient.WithNamespace(ops.NameSpace).List(metav1.ListOptions{LabelSelector:"kubernetes.io/hostname="+nodeName})
```

Comment:
This line has some cluster dependency(some platforms have nodeName and hostName are different ex: openshift new versions)